### PR TITLE
Trying to replace a bunch of code with foxyform

### PR DESCRIPTION
**Pull request by amastov**
I used this website called foxyform.com to automatically link this contact form to julian's studiocracy email.

### CODE REVIEW

**Comment by amastov** [comment]
Also just added the :confirmable module because I was getting a no method error when seeding database.

**Comment by amastov** [comment]
Here's what the form looks like:
images

File: app/models/user.rb [hunk 1]
```diff
@@ -18,9 +18,9 @@ class User < ActiveRecord::Base
 
 
   # Include default devise modules. Others available are:
-  # :confirmable, :lockable, :timeoutable and :omniauthable
+  # :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable, :registerable,
-  :recoverable, :rememberable, :trackable, :validatable
+  :recoverable, :rememberable, :trackable, :validatable, :confirmable
 # Confirmation email disabled due to timeout errors with our host
```
Comment (amastov) [inline app/models/user.rb]: Anyone want to elaborate on this? I couldn't perform a rake db:seed without the :confirmable module included.

**Comment by JCDJulian** [comment]
We actually used foxyform before. it's horrible. it secretly embeds popups into your site on mobile. we had to remove it which is why we're in the mess with the php mailer. not sure how to get it to work though.

the :confirmable was removed because we've been having problems with our hosting provider. our site's IP address is getting picked up by some blacklists for some reason. i need to sort that out before we can use comfirmable. in the mean time we just need to comment out all references to confirmability in devise, including skip_confirmation in the seeds

**Pull request closed without merging**